src/TLuaInterpreter.cpp

```diff
@@ -248,58 +248,105 @@ int TLuaInterpreter::Wait(lua_State* L)
     return 0;
 }
 
-// Documentation: ? - public function missing documentation in wiki
-QString TLuaInterpreter::dirToString(lua_State* L, int position)
+// Documentation: PRIVATE function with no documentation in wiki, converts an
+// exit direction number (in range 1 to 12) to corresponding text string used
+// as the key from them in room exit weight and custom exit line members of the
+// TRoom class. Note that a previous incarnation of this function returned the
+// complete lower case words in all cases which would break should those numbers
+// have been used - at present the Wiki does NOT mention the numbers which is
+// deliberate IIRC to prevent that borkage from showing... SlySven Nov 2018
```

Comment:
You mean putting my identifier there? It was to make it clear that the opinion (about the state of the Wiki for the public functions that use this undocumented {and not user accessible BTW} helper function not mentioning the use of *numbers* to specify a *normal* exit direction in `addCustomExit` and `setExitWeight` *because it **did not work!***) is entirely **my own** :imp:

(And may be in fact rong...)